app/views/responsible_body/devices/home/show.html.erb

```diff
@@ -1,68 +1,43 @@
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <%= title %>
+      </li>
+    </ol>
   </div>
 <% end %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">
-      <%= t('page_titles.responsible_body_devices_home') %>
-    </h1>
+    <h1 class="govuk-heading-xl"><%= title %></h1>
 
-    <h2 class="govuk-heading-m">Decide if schools can order their own devices</h2>
+    <h2 class="govuk-heading-l govuk-!-font-size-27">
+      <%= govuk_link_to t('page_titles.responsible_body_schools_list'), responsible_body_devices_schools_path %>
+    </h2>
 
-    <p class="govuk-body">You can allow schools to order their own devices. Or you can manage some, or all, orders centrally.</p>
+    <p class="govuk-body">Use this section to:</p>
 
-    <% if @responsible_body.is_a_trust? %>
-      <p class="govuk-body">If you let schools place orders you’ll need to nominate someone in each school for us to contact.</p>
-      <p class="govuk-body">Each contact will be able to:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>manage who can order devices</li>
-        <li>order devices themselves</li>
-      </ul>
-    <% else %>
-      <p class="govuk-body">We recommend letting schools place orders because it’s the quickest way to get help to those who need it.</p>
-      <p class="govuk-body">You’ll need to nominate someone in each school for us to contact.</p>
-
-      <p class="govuk-body">Each contact will be able to:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>manage who can order devices</li>
-        <li>order devices themselves</li>
-      </ul>
-
-      <h2 class="govuk-heading-m">Help schools prepare for disruption</h2>
-
-      <p class="govuk-body">Give us contact details as soon as you can so we can help schools prepare to deliver remote education if they need to.</p>
-    <% end %>
-
-    <h2 class="govuk-heading-m">Each school has their own allocation</h2>
-
-    <p class="govuk-body">Every school has its own provisional allocation of laptops and tablets. Allocations are based on free school meal data and estimates of how many devices a school has already received.</p>
-
-    <p class="govuk-body">Numbers will be reassessed at the time of ordering based on availability and the extent of coronavirus restrictions.</p>
-
-    <h2 class="govuk-heading-m">A school’s full allocation can only be ordered when local coronavirus restrictions are confirmed</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>tell us who will place orders for schools</li>
+      <li>give schools access to the services they need</li>
+      <li>see your list of schools and their allocations</li>
+    </ul>
 
-    <p class="govuk-body">We’ll contact people as soon as they’re able to place orders.</p>
+    <h2 class="govuk-heading-l govuk-!-font-size-27">
```

Comment:
the `govuk-!-font-size-27` is redundant

Reply:
It's not redundant. `27` is smaller than `l`

Reply:
Yep what @fofr said...